skyve-war/src/main/webapp/external/selfRegister.xhtml

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+		xmlns:ui="http://java.sun.com/jsf/facelets"
+		xmlns:h="http://java.sun.com/jsf/html"
+		xmlns:f="http://java.sun.com/jsf/core"
+		xmlns:p="http://primefaces.org/ui"
+		xmlns:pt="http://www.primefaces.org/ui"
+		xmlns:s="http://skyve.org/xml/ui"
+		dir="#{skyvePublic.dir}">
+	<f:view contentType="text/html" encoding="UTF-8">
+		<f:metadata>
+			<f:viewParam name="a" value="#{skyvePublic.webActionParameter}" />
+			<f:viewParam name="m" value="#{skyvePublic.bizModuleParameter}" />
+			<f:viewParam name="d" value="#{skyvePublic.bizDocumentParameter}" />
+			<f:viewParam name="q" value="#{skyvePublic.queryNameParameter}" />
+			<f:viewParam name="i" value="#{skyvePublic.bizIdParameter}" />
+			<f:viewParam name="c" value="#{skyvePublic.bizCustomerParameter}" />
+			<f:event type="preRenderView" listener="#{skyvePublic.preRender}"/>
+		</f:metadata>
+		<p:importEnum type="org.skyve.web.WebAction" var="WebAction" allSuffix="ALL_ENUM_VALUES" />  
+		<ui:composition template="#{'/WEB-INF/pages/templates/'.concat(skyvePublic.templateName).concat('/view.xhtml')}">
+			<ui:param name="bean" value="#{skyvePublic}" />
+			<ui:param name="showMenu" value="false" />
+			<ui:param name="resetMenu" value="false" />
+			<ui:param name="showSwitchMode" value="#{'phone' ne skyve.userAgentType and skyve.canSwitchMode}" />
+			<ui:param name="showLogout" value="false" />
+			<ui:define name="head">
+				<!-- Password strength estimator -->
+				<script src="https://cdnjs.cloudflare.com/ajax/libs/zxcvbn/4.2.0/zxcvbn.js"></script>
```

Comment:
Can you download this file and add it into Skyve where we have the other javascript files? That allows this to work offline and protects us from CDN poisoning attacks.